packages/quantic/force-app/main/default/lwc/quanticSourceCitations/quanticSourceCitations.html

```diff
@@ -0,0 +1,39 @@
+<template>
+  <template if:true={shouldDisplayCitations}>
+    <div class="source__citations slds-grid slds-align_vertical-center">
```

Comment:
Since this is no longer defined in the CSS.
```suggestion
    <div class="slds-grid slds-align_vertical-center">
```